server/db/model/User.js

```diff
@@ -0,0 +1,151 @@
+var bcrypt = require('bcrypt'),
+    jwt = require('jsonwebtoken'),
+    mongoose = require('../index.js'),
+    config = require('../../../config/default.js'),
+    Schema = mongoose.Schema,
+    secret = config.secret;
+
+// Define the document Schema
+var userSchema = new Schema({
+    full_name: String,
+    email: {
+        type: String,
+        required: true,
+        index: {
+            unique: true
+        }
+    },
+    email_verified: Boolean,
+    password: {
+        type: String,
+        required: true
+    },
+    tokens: [{
+        token: String
+    }],
+    old_tokens: [{
+        token: String
+    }],
+    created_date: {
+        type: Date,
+        default: Date.now
+    },
+    birthday: Date,
+    groups: [{
+        name: String
```

Comment:
Database groups. We could introduce the idea of `teams` as a `teams` property that is similar to this